docs/performanceprofile/performance_addon.md

```diff
@@ -2,7 +2,7 @@
 
 The `Performance Addon controller` optimizes OpenShift clusters for applications sensitive to cpu and network latency.
 
-![alt text](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/interactions/diagram.png "How Performance Addon controller interacts with other components and operators")
+!["How Performance Addon controller interacts with other components and operators"](https://github.com/openshift/cluster-node-tuning-operator/blob/master/docs/performanceprofile/interactions/diagram.png )
```

Comment:
Performance Addon Controller. I thin the lower case c is a typo from old commit.

Reply:
done